classes/external/get_data_merge_requests.php

```diff
@@ -90,45 +90,45 @@ public static function execute(string $removeuserfield,
                     keepuserfield, keepuservalue, keepuserid,
                     timeadded, timemodified, status, retries, log
                 FROM
-                   {".merge_request::TABLE_MERGE_REQUEST."}
+                   {" .merge_request::TABLE_MERGE_REQUEST. "}
                 ";
-        $params_query = array();
+        $paramsquery = array();
         if (isset($removeuserfield) && !empty($removeuserfield)) {
             $whereclauses[] = 'removeuserfield = ?';
-            array_push($params_query, $removeuserfield);
+            array_push($paramsquery, $removeuserfield);
         }
         if (isset($removeuservalue) && !empty($removeuservalue)) {
             $whereclauses[] = 'removeuservalue = ?';
-            array_push($params_query, $removeuservalue);
+            array_push($paramsquery, $removeuservalue);
         }
         if (isset($removeuserid) && !empty($removeuserid)) {
             $whereclauses[] = 'removeuserid = ?';
-            array_push($params_query, $removeuserid);
+            array_push($paramsquery, $removeuserid);
         }
         if (isset($keepuserfield) && !empty($keepuserfield)) {
             $whereclauses[] = 'keepuserfield = ?';
-            array_push($params_query, $keepuserfield);
+            array_push($paramsquery, $keepuserfield);
         }
         if (isset($keepuservalue) && !empty($keepuservalue)) {
             $whereclauses[] = 'keepuservalue = ?';
-            array_push($params_query, $keepuservalue);
+            array_push($paramsquery, $keepuservalue);
         }
         if (isset($keepuserid) && !empty($keepuserid)) {
             $whereclauses[] = 'keepuserid = ?';
-            array_push($params_query, $keepuserid);
+            array_push($paramsquery, $keepuserid);
         }
         if (isset($id) && !empty($id)) {
             $whereclauses[] = 'id = ?';
-            array_push($params_query, $id);
+            array_push($paramsquery, $id);
         }
         if (isset($status) && !empty($status)) {
             $whereclauses[] = 'status = ?';
-            array_push($params_query, $status);
+            array_push($paramsquery, $status);
         }
         if (count($whereclauses) > 0) {
             $sql .= 'WHERE ' . implode(' AND ', $whereclauses);
         }
-        return $DB->get_records_sql($sql, $params_query);
+        return $DB->get_records_sql($sql, $paramsquery);
```

Comment:
now I changed the code in a cleaner mode. The paramsquery contains only the parameters passed from postman.  In my case is an array with only one value (id=16).  
And sql is well formed. 
"SELECT id, removeuserfield, removeuservalue, removeuserid, keepuserfield, keepuservalue, keepuserid,                    timeadded, timemodified, status, retries, log FROM {tool_mergeusers_queue}  WHERE id = ?"
The problem is that when I execute the web service from postman I received the following error:
{"exception":"invalid_response_exception",
"errorcode":"invalidresponse",
"debuginfo":"Error in response - Missing following required key in a single structure: id"}